src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </section>
```

Comment:
There should be an empty line between multiline sibling blocks of HTML, but not between parent and child elements. Add an empty line between the closing </section> tag and the next <section> tag, but remove the empty line inside the <section> block.